app/views/events/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="flex_row">
+  <div>
+    <div class="when">
+      <%= l @event.moment, :format => :friendly %>
+    </div>
+
+    <h1 class="title">
+      <%= @event.title %>
+    </h1>
+    <p>
+      Hosted by <strong><%= link_to @event.microcosm.name, microcosm_path(@event.microcosm) %></strong>
+    </p>
+    <p>
+      Organized by: <strong>Andrew Wiseman</strong>
+    </p>
+  </div>
+  <div>
+    <p><%= @event.attendees.size %> people are going.</p>
+    <p>Are you going?</p>
+    <%= form_with :model => @my_attendance do |form| %>
+      <%= form.hidden_field(:event_id, :value => @event.id) %>
+      <%= form.hidden_field(:user_id, :value => current_user&.id) %>
+      <%= form.submit :value => "Yes" %>
```

Comment:
We'll need to change this so that what the user sees is via i18n, while the value submitted doesn't change